.eslintrc.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
maybe remove this file. You only need 1 file. I rather keep `.eslintrcc` than `.json`.

And this file JSON file contains configuration for React which is useless in a backend.

Also I guess `parserOptions` is deprecated in the ESLINT version 8.